scss/components/_Flash.scss

```diff
@@ -0,0 +1,35 @@
+// default component vars listed at the top of the component stylesheet
+// those vars are transferred to rev-settings AND are commented out
+// this way updates can be made to a component by uncommenting settings vars
+$flash-background: $primary !default;
+$flash-border-size: 1px !default;
+$flash-border-color: $divider-color-dark !default;
+$flash-color: $white !default;
+$flash-padding: $global-vertical-space $global-horizontal-space !default;
+
+.rev-Flash {
+  @include color-management;
+  @include typography-contained;
+  background: $flash-background;
+  border: $flash-border-size solid $flash-border-color;
+  color: $flash-color;
+  margin-bottom: $global-margin;
+  padding: $global-padding-tiny 0;
+  position: relative;
+
+  &.rev-Flash--fade {
+    @include fade-out--hide();
+  }
+  &.rev-Flash--dismissible {
+    cursor: pointer;
+  }
+}
+
+.rev-FlashClose {
+  background: none;
+  border: none;
+  color: $flash-color;
```

Comment:
this should probably get its own var. And will also need vars for `hover` (added to `:hover, :focus`) and `active` (added to `:active`)

`$flash-close-color`
`$flash-close-color-hover`
`$flash-close-color-active`

**And don't forget to add them to the settings-template**